lib/models/File.ts

```diff
@@ -4,6 +4,7 @@ export default class File {
   upload_metadata?: string
   upload_length?: string
   size?: number
+  creation_date?: Date
```

Comment:
Is this really optional? This property is set in constructor every time.

Reply:
Good point - I don't think this property should be optional.

Part of the reason I made `creation_date` optional was because there were a few places in `GCSDataStore` and `S3Store` that were initializing `File` objects with bracket notations:

<img width="351" alt="Screen Shot 2022-11-05 at 5 00 12 PM" src="https://user-images.githubusercontent.com/9089108/200147727-f16bbd61-2115-4952-8250-6da5d5438c62.png">

Making the creation_date required means that those places need to be defined with an appropriate value. At the time of creating this PR, I didn't want to make modifications outside of the `FileStore` file. However, since [your comment above](https://github.com/tus/tus-node-server/pull/320#issuecomment-1301192982) suggests looking into making the expiration code DataStore agnostic, I will also look into fixing those definitions of `File` objects when creation_date is marked as required.